### ISSUE TITLE
undefined symbol: mpv_detach_destroy

### ISSUE THREAD

**Issue by DocMAX** [jellyfin/jellyfin-mpv-shim#246]
new error on arch linux, everything updated.
using https://aur.archlinux.org/packages/jellyfin-mpv-shim/
```
docmax@game: ~ $ jellyfin-mpv-shim 
2021-12-25 11:18:41,496 [    INFO] conf: Loaded settings from json: /home/docmax/.config/jellyfin-mpv-shim/conf.json
Traceback (most recent call last):
  File "/usr/bin/jellyfin-mpv-shim", line 33, in <module>
    sys.exit(load_entry_point('jellyfin-mpv-shim==2.0.1', 'console_scripts', 'jellyfin-mpv-shim')())
  File "/usr/lib/python3.10/site-packages/jellyfin_mpv_shim/mpv_shim.py", line 65, in main
    from .player import playerManager
  File "/usr/lib/python3.10/site-packages/jellyfin_mpv_shim/player.py", line 42, in <module>
    import mpv
  File "/usr/lib/python3.10/site-packages/mpv.py", line 531, in <module>
    _handle_func('mpv_detach_destroy',          [],                                         None, errcheck=None)
  File "/usr/lib/python3.10/site-packages/mpv.py", line 478, in _handle_func
    func = getattr(backend, name)
  File "/usr/lib/python3.10/ctypes/__init__.py", line 387, in __getattr__
    func = self.__getitem__(name)
  File "/usr/lib/python3.10/ctypes/__init__.py", line 392, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))
AttributeError: /usr/lib/libmpv.so.2: undefined symbol: mpv_detach_destroy

```

**Comment by iwalton3** [comment]
I caught this when updating mpv for windows. It seems the dev version broke ABI. You could try using external MPV mode or downgrading to the latest release version of MPV.

**Comment by iwalton3** [comment]
Upstream issue with the python-mpv library: https://github.com/jaseg/python-mpv/issues/192

**Comment by DocMAX** [comment]
oh yeah, i didn't realize i was on GIT version. moved back to stable release.

**Comment by iwalton3** [comment]
https://github.com/jaseg/python-mpv released python-mpv 1.0 which I can confirm solves this issue.

Note that in the process it broke compatibility with older MPV versions so I'm not installing it by default.
